CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+3.0.0 March 29, 2017
```

Comment:
I'd like to follow [semver](http://semver.org/) for this project.  Could we make this `v2.1.0` instead?  This introduces new functionality in a backwards-compatible way.